src/mouse_events.cpp

```diff
@@ -240,40 +237,6 @@ void mouse_handler::touch_motion(int x, int y, const bool browse, bool update, m
 		
 		// we search if there is an attack possibility and where
 		attack_from = current_unit_attacks_from(new_hex);
-		
-		//see if we should show the normal cursor, the movement cursor, or
-		//the attack cursor
-		//If the cursor is on WAIT, we don't change it and let the setter
-		//of this state end it
-		if (cursor::get() != cursor::WAIT) {
```

Comment:
It's a touch UI handler. It doesn't need cursor at all, it's under the finger anyway. My intent was to leave cursor handling to mouse code, and leave only what was needed.